.evergreen/functions.yml

```diff
@@ -88,14 +88,15 @@ post:
       working_dir: src
       shell: bash
       script: |
-        tar czvf all-e2e-logs.tgz packages/compass-e2e-tests/.log
+        tar czf all-e2e-logs.tgz packages/compass-e2e-tests/.log
```

Comment:
This runs right at the end and was running with -v so every file was listed and recently a gazillion little .json files were added for each test's output..